tests/settings/settings.spec.test.ts

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const assert = require('assert').strict;
+import settingsMod from '../../backend/utils/Setting';
+import path from 'path';
+import process from 'process';
+import { describe, it, beforeEach } from 'vitest';
+
```

Comment:
what do you thinks about just Ising nodejs test-runner ?

Reply:
I guess we need vitest anyway because we also want to write tests for our React components.

Reply:
with nodejs runner normally it's possible using jsdom.

Reply:
What is faster?

Reply:
when I had work on nodejs.org, I can say nodejs is faster that jest. I'm not expert with vite. But What I know is nodejs is built-in and use maximum of cpu concurrently.

Reply:
Can it also handle the path aliases?

Reply:
YES it's done by `tsx` like in `server.ts`